storybook/stories/common/Widgets.stories.tsx

```diff
@@ -147,7 +147,7 @@ storiesOf('Common / Widgets', module)
     </div>
   ))
   // @ts-ignore ts-migrate(2345) FIXME: Argument of type '(props: {    locale: string;}) =... Remove this comment to see the full error message
```

Comment:
Would it be possible to remove / fix the `@ts-ignore`? Possibly that was the source of the runtime error? The comment is applicable to all `@ts-ignores` next to your changes.

Reply:
The error was the lack of passing any `props` down to any component, because Storybook expect the definition or acknowledgement of `args` as well. 
I will have a look at this.